### PR TITLE
fix extract_local_sensitivities

### PR DESCRIPTION
**Pull request by YingboMa**

### CODE REVIEW

File: src/local_sensitivity/forward_sensitivity.jl [hunk 1]
```diff
@@ -205,7 +205,15 @@ function extract_local_sensitivities(sol,::ForwardDiffSensitivity, ::Val{false})
   _sol = adapt(eltype(sol.u),sol)
   u = ForwardDiff.value.(_sol)
   du_full = ForwardDiff.partials.(_sol)
-  return u, [[du_full[i,j][k] for i in 1:size(du_full,1), j in 1:size(du_full,2)] for k in 1:length(du_full[1])]
+  @show size(du_full)
```
Comment (ChrisRackauckas) [inline src/local_sensitivity/forward_sensitivity.jl:208]:
```suggestion
```

File: src/local_sensitivity/forward_sensitivity.jl [hunk 2]
```diff
@@ -205,7 +205,15 @@ function extract_local_sensitivities(sol,::ForwardDiffSensitivity, ::Val{false})
...
+  firststate = first(du_full)
```
Comment (ChrisRackauckas) [inline src/local_sensitivity/forward_sensitivity.jl:209]:
```suggestion
```

File: src/local_sensitivity/forward_sensitivity.jl (hunk 1 above)
Comment (YingboMa) [reply src/local_sensitivity/forward_sensitivity.jl:208]: Oh haha

File: src/local_sensitivity/forward_sensitivity.jl (hunk 2 above)
Comment (YingboMa) [reply src/local_sensitivity/forward_sensitivity.jl:208]: `firststate` is actually used.

Comment (ChrisRackauckas) [reply src/local_sensitivity/forward_sensitivity.jl:208]: oh I see

**Pull request merged**